public/input_text_load_data.js

```diff
@@ -111,12 +111,10 @@ function loadData(collection) { //Called from top-level code
   else {
     collection = 'addr.json';
   }
-  console.log(collection);
 
   $.ajax({
     url: '/v1/search?pageLength=1000',
     data: {
-     // format: 'json',
       collection: collection
     },
     type: 'POST',
```

Comment:
Awesome. Good work Kevin. I would merge it but my guess is that @dmcassel will now that he is back